pkg/ffmpeg/encoder_screenshot.go

```diff
@@ -21,7 +21,9 @@ func (e *Encoder) Screenshot(probeResult VideoFile, options ScreenshotOptions) e
 		"-v", options.Verbosity,
 		"-ss", fmt.Sprintf("%v", options.Time),
 		"-y",
-		"-i", probeResult.Path, // TODO: Wrap in quotes?
+		// TODO: Wrap in quotes?, not required on linux/darwin. quotes are a shell level argument escape separator.
+		// Don't know about windows.
+		"-i", probeResult.Path,
```

Comment:
Yeah, pretty sure that TODO is no longer valid. Please remove the TODO altogether.

Reply:
Done. removed.